web/impact/impact/permissions/v1_api_permissions.py

```diff
@@ -75,7 +75,7 @@ def has_permission(self, request, view):
 
 class OfficeHourPermission(IsAuthenticated):
     def has_permission(self, request, view):
-        self.message = self.get_message(request)
+        message = self.get_message(request)
```

Comment:
this needs to be an attribute of the class, not a local variable of the function. see line 63